mapBuilder/locales/en_US/dictionary.UTF-8.properties

```
@@ -8,4 +8,5 @@ geobookmark.delete.confirm=Are you sure you want to delete this map?
 selector.layers.loading=Loading
 geobookmark.deleted=Map deleted.
 layer.added=Layer added.
-layer.deleted=Layer deleted.
\ No newline at end of file
+layer.deleted=Layer deleted.
+error.api.inkmap=API Key missing.
```

```suggestion
error.api.inkmap=API key missing.
```

Side note, but you can configure your IDE to always add an empty line at the end of file. It will avoid this red sign 🛑 https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file
useful when you do a `cat your_file` to avoid the prompt etc.